libs/ui/inputs/src/lib/dropdown-multiselect/dropdown-multiselect.component.html

```diff
@@ -64,7 +66,7 @@
         class="max-w-full bg-main text-white rounded pr-[7px] pl-2.5 flex gap-2 items-center opacity-70 hover:opacity-100 focus:opacity-100 transition-opacity"
         (click)="select(selected, false)"
       >
-        <div class="text-sm truncate leading-[19px]">{{ selected.label }}</div>
+        <div class="text-sm truncate leading-[26px]">{{ selected.label }}</div>
```

Comment:
why ?
to me it's not well aligned with this value
![Screenshot from 2022-12-15 14-58-53](https://user-images.githubusercontent.com/1491924/207879565-e7021b4c-c325-4da2-8448-670fca3db08f.png)

Reply:
Try to select enough values to wrap to multiple lines ;-)

Reply:
it's the line-height of the label in the tag.
to me it's not aligned with the cross circle. it's not about the amount of selected value in the input is it ?

Reply:
Maybe it should be fixed differently, but yes if you select a high amount of values for the input, so they are wrapped over multiple lines, the badges were stashed together with smaller line height. As long as they stay in one line I actually do not see a difference in the alignment.